src/Error/GraphQLException.php

```diff
@@ -181,6 +207,32 @@ public function getOriginalErrorMessage(): ?string
         return null !== $this->originalException ? $this->originalException->getMessage() : null;
     }
 
+    /**
+     * @inheritdoc
+     */
+    public function toArray(): array
+    {
+        $result = [
+            'message'    => $this->message,
+            'locations'  => $this->getLocationsAsArray(),
+            'path'       => $this->path,
+        ];
+
+        if (null !== $this->extensions) {
+            $result['extensions'] = $this->extensions;
```

Comment:
Can you update the tests for this method to check that this extensions handling works correctly?

Reply:
I added tests for this and some other things.